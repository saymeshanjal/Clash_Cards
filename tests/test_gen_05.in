10
14 3 1
10 39 1
4 17 4
8 47 5
12 30 4
16 3 3
6 21 3
8 28 3
12 20 3
8 31 4