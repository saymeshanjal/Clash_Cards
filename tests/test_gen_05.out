1254 2178 0
126 126 0
0 5 1
8 62 0
14 684 226
1287 11583 0
6 14 0
27 43 0
112 812 0
8 42 20
